.editorconfig

```diff
@@ -0,0 +1,148 @@
+root = true
+
+[*.cs]
+indent_style=space
+indent_size=2
+tab_width=2
+end_of_line=lf
+charset=utf-8
+trim_trailing_whitespace=true
```

Comment:
This one can be put everywhere as well as :
```
end_of_line=lf
charset=utf-8
```

Reply:
done